integration-tests/python/satellite/open_stream_test.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 Infostellar, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The integration tests for OpenSatelliteStream API in StellarStationService."""
+
+from queue import Queue
+
+from stellarstation.api.v1 import stellarstation_pb2
+
+SATELLITE_ID = '98'
+
+
+# This test checks the expected status after sending status change command.
+def test_open_satellite_stream(stub_factory):
+    client = stub_factory.get_satellite_service_stub()
+
+    request_queue = Queue()
+    request_iterator = generate_request(request_queue)
+
+    expected_status = -1
+    for response in client.OpenSatelliteStream(request_iterator):
+        if response.HasField("receive_telemetry_response"):
+            telemetry_data = response.receive_telemetry_response.telemetry.data
+            assert len(telemetry_data) > 1
+
+            # The second last byte of the telemetry indicates the current status of the fake satellite used
+            # in the test. The value is either of 0 or 1.
```

Comment:
Is that something you should also assert? (That the value is 0 or 1)?

Reply:
Added assert to check the value.